pages/ppr-marketing.vue

```diff
@@ -539,11 +499,45 @@ a {
   border-bottom: 1px dotted $gray9;
 }
 
```

Comment:
When using media queries to effect breakpoints (as below), I have found it much easier to work from small (mobile) to larger. That's why I use the `min-width` selector instead of the `max-width` selector. And mixing them is just hell 😝 

Also, since we're using a mixture of Vuetify breakpoints (for columns) (which are equivalent to Bootstrap breakpoints, by the way), it's way easier to just use those breakpoints instead of declaring our own (as that would again mix breakpoints).